### PR TITLE
Update 11-performance.md

### PR DESCRIPTION
**Pull request by ErinBecker**
This PR updates the episode 11-performance.md, including: 

1) Updating questions and keypoints to reflect current content.

2) Rewriting the introductory paragraph to emphasize how this episode fits in with our practice/feedback pedagogical model.

3) Significantly shortening the section on lesson study and refocusing to emphasize how the Carpentries supports lesson study.

4) Removing learning sideways (see [this issue](https://github.com/ErinBecker/instructor-training-reorg/issues/173))

5) Adding preparation time to instructions for practice teaching exercise.

6) Removing section on body language.

7) Adding summary paragraph tying this material in with growth mindset and practice/feedback again.

### CODE REVIEW

**Comment by ErinBecker** [comment]
8) Changing title to show parallel with episode 02-practice as per issue https://github.com/ErinBecker/instructor-training-reorg/issues/3

**Comment by ErinBecker** [comment]
Fixes https://github.com/ErinBecker/instructor-training-reorg/issues/3

**Comment by ErinBecker** [comment]
@ChristinaLK please let me know if this is ok to merge once you've had a chance to take a look.

File: _episodes/11-performance.md [hunk 1]
```diff
@@ -96,48 +63,50 @@ the school board,
...
 > 1.  Split into groups of three.
-> 2.  Have each person introduce themselves
->     and then explain, in no more than 90 seconds,
->     the key idea or ideas from
->     the Carpentry lesson episode they chose before the start of the training course
->     to another person in the group
->     while the third person records it (video and audio)
+> 2.  Individually, spend 5 minutes preparing to teach a 90-second segment of 
+>     the Carpentry lesson episode you chose before the start of the training course.
+> 3.  Get together with your group and have each person teach their segment to the group,
```
Comment (ChristinaLK) [inline _episodes/11-performance.md]: maybe clarify that they *shouldn't* live code?

**Comment by ErinBecker** [comment]
Thanks @ChristinaLK. I've made the recommended edit. Merging on @ChristinaLK's approval.

**Pull request merged**